azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/configurations/DeploymentType.java

```diff
@@ -29,17 +29,17 @@
 
     public static DeploymentType fromString(final String input) {
         if (StringUtils.isEmpty(input)) {
-            return MS_DEPLOY;
+            return ZIP;
```

Comment:
Here to reconsider the fallback logic. When users use this setting entry but with empty or wrong values, we should explicitly show an error but not recover silently. The current logic tolerant wrong input and the wrong configuration will stay without anyone trying to fix because they don't know there's an error.